README.md

```diff
@@ -206,6 +219,21 @@ d.save()
 
 Configuring the plugin this way might be useful if you're running your Jenkins Master in a Docker container using the [Official Jenkins Docker Image](https://github.com/jenkinsci/docker) or any derivative that supports plugins.txt and Groovy init scripts.
 
+### Configure with an environment variables
+
+You have the option of configuring your Datadog plugin using environment variables. 
+You will have to specify three variables:
+- `DATADOG_JENKINS_PLUGIN_REPORT_WITH` specify which authentication mechanism you want to use. When set to `DSD` it will use a DogStatsD Server to report to Datadog. Otherwise set it to the default `HTTP` value.
```

Comment:
```suggestion
- `DATADOG_JENKINS_PLUGIN_REPORT_WITH` specify which report mechanism you want to use. When set to `DSD` it will use a DogStatsD Server to report to Datadog. Otherwise set it to the default `HTTP` value.
```